README.md

```diff
@@ -10,19 +10,19 @@ A Bitcoin transaction consists of inputs and outputs. [CoinJoins](https://bitcoi
 
 Furthermore notice that both JoinMarket and SharedCoin could use the constructions of Chaumian CoinJoin or CoinShuffle to facilitate unlinkability. However in case of SharedCoin, if it would use any of these constructions, then it would be equivalent to these constructions, while in case of JoinMarket, since the linkability is mitigated in other means, the complexity may not be justified.
 
-In this paper we build upon Chaumian CoinJoins. While Chaumian CoinJoins guarantee unlinkability through changing anonymity network identities, it also utilizes a blind signature scheme to prevent denial of service attacks. However in its current form it only works, because the blind signatures provided by the coordinator also correspond with pre-defined denominations. This works for self mixing, because anonymity is provided by making outputs equal, however it prohibits creating arbitrary output amounts, which is essential for making payments in coinjoins. In such cases anonymity can be achieved in different ways, like making the inputs equal, mixing on the change output or through utilizing [Knapsack Mixing](https://www.comsys.rwth-aachen.de/fileadmin/papers/2017/2017-maurer-trustcom-coinjoin.pdf). Furthermore such scheme would also enable unlinkability on all inputs of a participant and gain computational privacy against [subset sum attacks](https://en.wikipedia.org/wiki/Subset_sum_problem) by the coordinator. Another issue that comes up with Chaumian CoinJoins is the consolidations of many UTXOs it creates. Solving the provlem also enables consolidating them in a private way, at least on the transaction construction level.
+In this paper we build upon Chaumian CoinJoins. While Chaumian CoinJoins guarantee unlinkability through changing anonymity network identities, it also utilizes a blind signature scheme to prevent denial of service attacks. However in its current form it only works, because the blind signatures provided by the coordinator also correspond with pre-defined denominations. This works for self mixing, because anonymity is provided by making outputs equal, however it prohibits creating arbitrary output amounts, which is essential for making payments in CoinJoins. In such cases anonymity can be achieved in different ways, like making the inputs equal, mixing on the change output or through utilizing Knapsack Mixing[REFERENCE]. Furthermore such scheme would also enable unlinkability on all inputs of a participant and gain computational privacy against subset sum attacks by the coordinator[REFERENCE]. Another issue that comes up with Chaumian CoinJoins is the consolidations of many UTXOs it creates. Solving the provlem also enables consolidating them in a private way, at least on the transaction construction level.
```

Comment:
```suggestion
In this paper we build upon Chaumian CoinJoins. While Chaumian CoinJoins guarantee unlinkability through changing anonymity network identities, it also utilizes a blind signature scheme to prevent denial of service attacks. However in its current form it only works, because the blind signatures provided by the coordinator also correspond with pre-defined denominations. This works for self mixing, because anonymity is provided by making outputs equal, however it prohibits creating arbitrary output amounts, which is essential for making payments in coinjoins. In such cases anonymity can be achieved in different ways, like making the inputs equal, mixing on the change output or through utilizing [Knapsack Mixing](https://www.comsys.rwth-aachen.de/fileadmin/papers/2017/2017-maurer-trustcom-coinjoin.pdf). Furthermore such scheme would also enable unlinkability on all inputs of a participant and gain computational privacy against [subset sum attacks](https://en.wikipedia.org/wiki/Subset_sum_problem) by the coordinator. Another issue that comes up with Chaumian CoinJoins is the consolidations of many UTXOs it creates. Solving the problem also enables consolidating them in a private way, at least on the transaction construction level.
```